lib/matplotlib/text.py

```diff
@@ -1251,7 +1251,8 @@ def set_verticalalignment(self, align):
 
         Parameters
         ----------
-        align : {'bottom', 'baseline', 'center', 'center_baseline', 'top'}
+        align : {'bottom', 'baseline', 'center', 'center_baseline', 'top'}, \
+default: baseline
         """
```

Comment:
same as above: there is now default value here as the `align` parameter is not optional.